Aquality.Selenium.Template/Aquality.Selenium.Template.NUnit/Tests/CustomImageComparatorTest.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Aquality.Selenium.Template.NUnit.Constants;
+using Aquality.Selenium.Template.Utilities;
+using NUnit.Framework;
+using System.Drawing;
+
+namespace Aquality.Selenium.Template.NUnit.Tests
+{
+    public class CustomImageComparatorTest : BaseTest
+    {
+        const float TheThresholdValueOf10Percent = 0.1f;
+
+        [Test(Description = "TC-0005 Comparison of two given images")]
+        public void TC0005_ComparisonOfTwoGivenImages()
+        {
+            var firstImage = Image.FromFile(ResourceConstants.FirstTestImage);
+
+            var firstImageWidth = firstImage.Width;
+
+            var firstImageHeight = firstImage.Height;
+
+            var secondImage = Image.FromFile(ResourceConstants.SecondTestImage);
+
```

Comment:
We should not do file reading and assertions in the test method. We should do them in steps, so that we could  simply define a problem cause from the stacktrace.

Reply:
The code was updated.